client/flow_quiet.py

```diff
@@ -12,11 +12,30 @@
 # Does an OIDC flow and finishes with a fetch of the userinfo endpoint
 #
 
-
-create_code_wait_server()
+print('Generating', sys.argv[1], 'visa')
 
 s = requests.Session()
 
+createFixture = s.post(
+    f"{CONTROL}/create",
+    data={
+        "scenario": "2022",
+        "forceSubject": sys.argv[1],
+        "forceAccept": True
+    },
+)
+
+fixtureId = createFixture.json()['id']
+
+setFixture = s.get(
+    f"{ISSUER}/setFixture",
+    params={
+        "fixtureId": fixtureId
+    }
+)
```

Comment:
Not quite sure what the purpose of setFixture is?
(I'm thinking it is a technique to tell the broker the current fixture context? If so, that's fine but note that that is only needed for the localhost dev deployment).

For the real deployment, I was imagining the fixture context is built from the URL used.

i.e. if I get a fixture id of `abcdefg` then the broker becomes

`broker-abcdefg.test.agha.org.au` (say)

and that automatically gives the fixture context to the broker backend (derived from the URL itself)

Reply:
Sorry for such a late reply, yes I think the real deployment makes a lot of sense. Just when running though, it was a bit of a pain to be able to generate a new fixture and use that fixture to log in with. Will update docs to specify this.